### PR TITLE
Some linting

### PR DESCRIPTION
**Pull request by anupvarghese**

### CODE REVIEW

**Comment by danielmoi** [comment]
Was this using a linter?

**Comment by shavz** [comment]
> Was this using a linter?

Should be using rustformat

**Comment by anupvarghese** [comment]
rustfmt & vscode, just tells all possible corrections and refactorings

File: src/lib.rs [hunk 1]
```diff
@@ -148,10 +133,10 @@ impl<'a> RRule<'a> {
         let mut until = "";
 
         // assign default weekstart and reassign if present
-        let mut wkst = "MO";
+        let mut _wkst = "MO";
```
Comment (shavz) [inline src/lib.rs]: I plan to eventually add support for this, but for now this is right

Comment (anupvarghese) [reply src/lib.rs]: yeah, we can re-enable that at that time.

**Pull request merged**